build.rs

```diff
@@ -72,13 +72,18 @@ fn main() {
 
   // Early exit
   if is_cargo_doc || is_rls {
+    print_prebuilt_src_binding_path();
     return;
   }
 
   print_link_flags();
 
   // Don't attempt rebuild but link
   if is_trybuild {
+    println!(
+      "cargo:rustc-env=RUSTY_V8_SRC_BINDING_PATH={}",
+      env::var("RUSTY_V8_SRC_BINDING_PATH").unwrap()
```

Comment:
Does this env var needs to be set up when building locally?

Reply:
nope